src/clj/game/cards/resources.clj

```diff
@@ -1139,6 +1139,17 @@
                  :effect (effect (derez target)
                                  (trash card {:cause :ability-cost}))}]}
 
+   "Miss Bones"
+   {:data {:counter {:credit 12}}
+    :implementation "Credit use restriction not enforced"
+    :abilities [{:counter-cost [:credit 1]
+                 :msg "gain 1 [Credits] for trashing installed cards"
+                 :async true
+                 :effect (req (gain-credits state :runner 1)
+                              (if (zero? (get-counters (get-card state card) :credit))
+                                (trash state :runner eid card {:unpreventable true})
```

Comment:
If you pass in eid to trash you shouldn't need effect completed right?

Reply:
Correct, that's my understanding. Passing `eid` to `trash` makes it responsible for completing the effect. The other branch of the `if` statement, I'm still responsible for calling `effect-completed`.

Reply:
Ah, I missed this was different branches of the `if` statement.